react/week2/to-do-app/src/ToDo.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+function ToDo(props){
+    const [enabled, setEnabled] = useState(false);
+    const handleChange = (id) => {
+        setEnabled(prevState => !prevState);
+        const style=enabled?'none':'line-through';
+        const labelElement=document.getElementById('label'+id);
+        if(labelElement){
+            labelElement.style.textDecorationLine=style;
+        }
```

Comment:
This should definitely work, but it is a little bit like using just plain HTML, CSS and JS. When we are using React, we can do these kinds of conditional styling in a slightly cleaner way (see my comment below).

Reply:
Thank you- I have changed the code accordingly